config/environments/development.rb

```diff
@@ -40,4 +40,14 @@
 
 
   config.action_mailer.default_url_options = { host: 'thingr.local' }
+
+  config.paperclip_defaults = {
+    :storage => :fog,
+    :fog_credentials => {
+      :provider => "AWS",
+      :aws_access_key_id => ENV['AWS_ACCESS_KEY_ID'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.